### ISSUE TITLE
Removing Beardedspice destroys default media key behaviour. 

### ISSUE THREAD

**Issue by FreeeG** [beardedspice/beardedspice#958]
1. Installed beardedspice through homebrew. `brew cask install beardedspice`
2. Realized that beardedspice did not do what I wanted it to do
3. Uninstalled beardedspice through homebrew

Now my media keys wont work at all. 

--
Running MacOS 10.15.7
Using external apple magic keyboard 2 w/numpad

**Comment by cwouter** [comment]
I've had the same issue. Have you found a way to reset the media keys manually?

**Comment by cwouter** [comment]
I've installed [macmediakeyforwarder](https://github.com/milgra/macmediakeyforwarder/releases), and that seems to fix the problem.

**Comment by FreeeG** [comment]
@cwouter Did you get the macmediakeyforwarder to work? I used that for a long time - but since the package is no longer maintained and it stopped working on my computer I started to look for alternatives. Turns out that Beardedspice was not the solution for me after all. It did more harm than good 😅

**Comment by cwouter** [comment]
@FreeeG , yes it works just fine. I am on OSX 10.15.4 currently. It opens Spotify instead of iTunes when I click the play button on my external keyboard.

**Comment by FreeeG** [comment]
@cwouter Wierd. Just installed it on another computer (running 10.15.7) and it does not work. It just instant crashes and is listed in red and (Not Responding) under the Activity Monitor

**Comment by cwouter** [comment]
Are you using the same version 3.1 (build 1)?
![image](https://user-images.githubusercontent.com/1439383/101501524-241e0680-3970-11eb-8fb1-530a8c11f970.png)

**Comment by FreeeG** [comment]
@cwouter I would assume so. It is 3.1 but not sure about the build as it won't start at all on my computer. Downloaded it from [here](https://github.com/milgra/macmediakeyforwarder/releases/tag/3.1)

<img width="830" alt="Screenshot 2020-12-08 at 16 48 37" src="https://user-images.githubusercontent.com/5333659/101506007-3a7a9100-3975-11eb-8373-f839f4b30666.png">

**Comment by xavi-dev** [comment]
Check https://github.com/beardedspice/beardedspice/issues/925 
This worked for me
